eng/Versions.props

```diff
@@ -88,9 +88,9 @@
     <MicrosoftBuildLocatorPackageVersion>1.2.6</MicrosoftBuildLocatorPackageVersion>
     <MicrosoftBuildPackageVersion>15.8.166</MicrosoftBuildPackageVersion>
     <MicrosoftBuildUtilitiesCorePackageVersion>15.8.166</MicrosoftBuildUtilitiesCorePackageVersion>
-    <MicrosoftCodeAnalysisCommonPackageVersion>3.3.0</MicrosoftCodeAnalysisCommonPackageVersion>
-    <MicrosoftCodeAnalysisCSharpPackageVersion>3.3.0</MicrosoftCodeAnalysisCSharpPackageVersion>
-    <MicrosoftCodeAnalysisWorkspacesCommonPackageVersion>3.3.0</MicrosoftCodeAnalysisWorkspacesCommonPackageVersion>
+    <MicrosoftCodeAnalysisCommonPackageVersion>3.4.0</MicrosoftCodeAnalysisCommonPackageVersion>
```

Comment:
Changing these will impact the runtime compilation package. Would it be possible to not change these and only change the VSIX variants? If not verify with @pranavkm to make sure that's ok.

That's the reasoning for the VSIX versions so at the tooling layers we can depend on newer versions

Reply:
Totally missed your reply on this PR.

> Would it be possible to not change these and only change the VSIX variants?

Nope. It didn't work because of multiple version conflicts.

Reply:
This seems fine.